apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -80,11 +80,44 @@ for single_test_env in ${test_env//,/ }; do
   passthrough_env+=("SIMCTL_CHILD_$key=$value")
 done
 
+xcrun_target_app_path=""
+xcrun_is_xctrunner_hosted_bundle="false"
+xcrun_is_ui_test_bundle="false"
 if [[ -n "$test_host_path" ]]; then
   xctestrun_test_host_path="__TESTROOT__/$test_host_name.app"
   xctestrun_test_host_based=true
   # If this is set in the case there is no test host, some tests hang indefinitely
   xctestrun_env+="<key>XCInjectBundleInto</key><string>$(escape "__TESTHOST__/$test_host_name.app/$test_host_name")</string>"
+
+  ui_testing_enabled="%(ui_testing_enabled)s"
+  if [[ -n "$ui_testing_enabled" ]]; then
```

Comment:
I think instead of doing this and requiring the separate runner we can use the `test_type` expansion? https://github.com/bazelbuild/rules_apple/blob/e6a22d7017813bc1825b3e5fbeb525311f4d9cf8/apple/internal/testing/apple_test_rule_support.bzl#L117 then we can just reuse the ordered runner? (maybe we should just rename that to have a more default name)

Reply:
Will do

Reply:
Looks like I always get `XCTEST` from that variable

Reply:
hrm, definitely should give you something else https://github.com/bazelbuild/rules_apple/blob/e6a22d7017813bc1825b3e5fbeb525311f4d9cf8/apple/internal/testing/ios_rules.bzl#L52